### PR TITLE
CADC-9017: make VOSpace storage configurable

### PR DESCRIPTION
**Pull request by hjeeves**
by passing in resourceID uri for storage service, service name and node URI prefix (resourceID) via org.opencadc.vosui.properties file.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -13,7 +13,7 @@ repositories {
 
 sourceCompatibility = '1.8'
 group = 'org.opencadc'
-version = '1.1.46'
+version = '1.1.47'
```
Comment (at88mph) [inline build.gradle:16]: Please make this version `1.2.0` as it is a big enough change (non backwards compatible configuration).

**Review by at88mph — CHANGES_REQUESTED** [review]

File: org.opencadc.vosui.properties [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+#org.opencadc.vosui.service.name = vault
+
+# The resource id of the VOSpace web service to use
+#org.opencadc.vosui.vault.service.resourceid = ivo://cadc.nrc.ca/vault
+
+# Base URI to use as node identifier
+#org.opencadc.vosui.vault.node.resourceid = vos://cadc.nrc.ca!vault
+
+# vault-related values
+org.opencadc.vosui.service.name = cavern
```
[the remark this answers is not in the document]
Comment (hjeeves) [reply org.opencadc.vosui.properties]: ok

File: src/main/java/ca/nrc/cadc/beacon/web/restlet/StorageApplication.java [hunk 3]
```diff
@@ -146,13 +155,31 @@ public StorageApplication(Context context) {
     @Override
     public Restlet createInboundRoot() {
         final Context context = getContext();
+        log.debug("context: " + context);
+
+        // Read properties file org.opencadc.vosui.properties
+        PropertiesReader propReader = new PropertiesReader("org.opencadc.vosui.properties");
```
[the remark this answers is not in the document]
Comment (hjeeves) [reply src/main/java/ca/nrc/cadc/beacon/web/restlet/StorageApplication.java]: Switched over to ApplicationConfiguration.java

File: build.gradle (hunk 1 above)
Comment (hjeeves) [reply build.gradle]: ok

**Review by at88mph — APPROVED** [review]

**Pull request merged**